data/nav.yml

```diff
@@ -0,0 +1,7 @@
+nav_items:
+  - title: "About"
+    link: "/about"
+  - title: "CFP"
+    link: "/#cfp"
```

Comment:
Yeah. Papercall has the most comprehensive info, and it puts them in the context of submitting (rather than making them click through to submit).